lib/react_on_rails/helper.rb

```diff
@@ -556,8 +556,7 @@ def initialize_redux_stores
 
     def raise_generated_missing_pack_warning(component_name)
       msg = <<~MSG
-        **ERROR** ReactOnRails: Generated missing pack for Component: #{component_name}. Please refresh the webpage \
-        once webpack has finished generating the bundles. If the problem persists
+        **ERROR** ReactOnRails: Generated missing pack for Component: #{component_name}. Please restart the webpack dev server to ensure webpack generates the chunks corresponding to #{component_name} component. If the problem persists
```

Comment:
or webpack in watch mode

Reply:
We can not run webpack in watch mode as creating a new Component makes a new entry point. To accommodate this new entry point, the server needs to get restarted as setEntry to webpack is configured on initialization of webpack.

Reply:
that's my point

the error says webpack dev server only

Reply:
Fixed